content/en/docs/_index.md

```diff
@@ -4,28 +4,9 @@ linkTitle: Docs
 menu: { main: { weight: 10 } }
 ---
 
-OpenTelemetry is a set of APIs, SDKs, tooling and integrations that are designed
-for the creation and management of _telemetry data_ such as traces, metrics, and
-logs. The OpenTelemetry documentation is intended to broadly cover key terms,
-concepts, and instructions on how to use OpenTelemetry in your software.
+OpenTelemetry, also known as OTel for short, is a vendor-neutral open-source [Observability](concepts/observability-primer/#what-is-observability) framework for instrumenting, generating, collecting, and exporting telemetry data ([traces](concepts/observability-primer/#distributed-traces), [metrics](concepts/observability-primer/#reliability--metrics), [logs](concepts/observability-primer/#logs)). As an industry-standard it is [natively supported by a number of vendors](/vendors)
```

Comment:
Can we keep the "such as" for traces, metrics, logs if we ever plan to have additional signals.

Reply:
Makes sense - will do!